libs/ftu-ui-components/src/lib/molecules/src/lib/source-list/source-list.component.scss

```diff
@@ -78,6 +79,10 @@
         overflow: hidden;
         text-overflow: ellipsis;
         white-space: nowrap;
+
+        span {
```

Comment:
Have you tried setting this using `mat.table-overrides` in the `hra-table` selector?